### PR TITLE
Feature/empty file icon

### PR DESCRIPTION
**Pull request by santiagonoguez**
## Description
Style guide now defined a non-decorated Screeny icon when no images are present. Also some styles of the previous empty screen page on both Video and Image were not matching the Style guide.

Also, styles for both file list and storage were almost identical and duplicated for a very similar page. So I combined both in a single generic style, and applied the changes to those.

Related PR: https://github.com/Rise-Vision/rise-vision-apps/pull/1168

## Motivation and Context
Complete storage implementation of vide.

## How Has This Been Tested?
Can be tested here for both video lists and storage ( captures below ):
https://apps-stage-10.risevision.com/templates/edit/008293db-6ec4-47cc-8908-94f8e4758d9e/?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443

Can be tested here for image lists:
https://apps-stage-10.risevision.com/templates/edit/0f5e691d-6291-4081-8f79-d0f9da3b8cf5/?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443

storage

videos

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
    - Manual checks above, but no easy way to automate changes as it is visual.
    - Will deploy on Monday, and verify behavior on Production.
    - Simple rollback, as it affects no data
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No documentation changes needed. No need to notify support.

### CODE REVIEW

File: src/scss/sections/template-editor-layout.scss [hunk 1]
```diff
@@ -638,32 +638,33 @@ $madero-light-blue: #107DDA;
...
+    padding: 0;
+    margin-top: 0;
+  }
 
-    img {
-      margin: 0 auto;
-    }
+  p {
+    font-size: 12px !important;
+    line-height: 20px;
```
Comment (willread) [inline src/scss/sections/template-editor-layout.scss]: It looks like this is 18px in the [style guide](https://app.goabstract.com/projects/edc4cf60-70e9-11e9-8aa3-ddcb2624e72e/branches/master/collections/16479415-04a9-4f9e-aa8a-4b1a96851543?collectionLayerId=9670fc0f-d0c7-4ac4-b166-08b55074714f&mode=build&selected=644321871-C05F53D4-163A-4507-A111-B1BD08C242E4)

File: src/scss/sections/template-editor-layout.scss [hunk 2]
```diff
@@ -638,32 +638,33 @@ $madero-light-blue: #107DDA;
...
+    padding-bottom: 20px;
```
Comment (willread) [inline src/scss/sections/template-editor-layout.scss]: The whitespace between the bottom of the <p> and the top of the image looks like about 38px total in [the styleguide](https://app.goabstract.com/projects/edc4cf60-70e9-11e9-8aa3-ddcb2624e72e/branches/master/collections/16479415-04a9-4f9e-aa8a-4b1a96851543?collectionLayerId=9670fc0f-d0c7-4ac4-b166-08b55074714f&mode=build&selected=442505580-B42C19DB-B440-4E0B-A7E7-0FE3980451C8), I think you could add some extra padding here.

File: src/scss/sections/template-editor-layout.scss (hunk 1 above)
Comment (willread) [inline src/scss/sections/template-editor-layout.scss]: It looks like this is 18px in the [style guide](https://app.goabstract.com/projects/edc4cf60-70e9-11e9-8aa3-ddcb2624e72e/branches/master/collections/16479415-04a9-4f9e-aa8a-4b1a96851543?collectionLayerId=9670fc0f-d0c7-4ac4-b166-08b55074714f&mode=build&selected=644321871-C05F53D4-163A-4507-A111-B1BD08C242E4)

File: src/scss/sections/template-editor-layout.scss (hunk 2 above)
Comment (santiagonoguez) [reply src/scss/sections/template-editor-layout.scss]: @willread  it's 43 px total, and as we already have 10px margin, I've updated this to be 33px, thanks !

File: src/scss/sections/template-editor-layout.scss (hunk 1 above)
Comment (santiagonoguez) [reply src/scss/sections/template-editor-layout.scss]: @willread updated, thanks !

**Pull request merged**